### ISSUE TITLE
Microsoft Defender reports a trojan in the latest update

### ISSUE THREAD

**Issue by fcnjd** [nvda-es/NVDARemote#47]
Hi,
at first, thank you for this useful add-on, it definitely brings a number of valuable improvements to the existing NVDA remote.
The NVDA store shows an update for TeleNVDA, however the download constantly fails since Microsoft Defender reports the following threat in it and therefore puts the file directly into quarantene, keeping NVDA from installing it: `Trojan:Script/Wacatac.B!ml`
Sure, this can be prevented by temporarily disabling Microsoft Defender - however in the long run, it would likely be better if this add-on wouldn't be flagged at all. Can you please check for that? Thanks in advance.

**Comment by jmdaweb** [comment]
Hi,
See release notes of latest updates for more information. The false positive, url_handler.exe, has been reported to multiple anti-virus software providers with no luck. There's nothing we can do to fix this beyond warning users.
Regards.

**Comment by fcnjd** [comment]
Thank you for your quick clarification. Sorry for that issue then, I checked the existing issues - but not the release comments. That was my mistake.
